packages/sources/src/addSourceWizard/hardcodedSchemas.js

```diff
@@ -213,7 +213,10 @@ export default {
                         }, {
                             name: 'credentials.subscription_id',
                             component: componentTypes.TEXT_FIELD,
-                            label: 'Subscription ID'
+                            label: 'Subscription ID',
+                            validate: [{
```

Comment:
This is already declared here https://github.com/RedHatInsights/frontend-components/blob/854bb9a00b2d536bb3c69c712ec4f79120785d91/packages/sources/src/addSourceWizard/hardcodedSchemas.js#L153

Reply:
Thanks! I missed that.